src/writer/serve.py

```
@@ -146,6 +155,75 @@
+        if self.counter_data is None:
+            # If the counter does not exist, initialize it to 0
+            self.counter = self.api_client.set_value(self.counter_key, 0)
+        else:
+            self.counter = self.counter_data
+
+    def _increment_counter(self):
+        try:
+            self.counter += 1
+            self.api_client.set_value(self.counter_key, self.counter)
+        except Exception as e:
+            logging.error(f"Failed to increment job counter: {e}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)